src/libs/KeyboardShortcut/index.js

```diff
@@ -18,7 +18,7 @@ const documentedShortcuts = {};
  * @returns {Array}
  */
 function getDocumentedShortcuts() {
-    return _.values(documentedShortcuts);
+    return _.sortBy(_.values(documentedShortcuts), 'displayName');
```

Comment:
actually... I think `sortBy()` works on objects so there's no need to call `values()`

Reply:
I just tested and saw this:

![2023-05-22_14-54-01](https://github.com/Expensify/App/assets/32969087/4e50bd85-0b8d-4ad3-8d49-9b98f335fbd5)